udata_gouvfr/views/site.py

```diff
@@ -83,7 +83,7 @@ def home():
     context = {
         'spd_datasets': Dataset.objects.filter(badges__kind='spd'),
         'recent_datasets': Dataset.objects.visible(),
-        'recent_reuses': Reuse.objects(featured=True).visible(),
```

Comment:
It's still tricky IMHO, even more since we won't see them (who goes on HP and check every tab every day ?). ATM I read the digest every day and ~ 1 in 2 new reuse is a spam.

Reply:
<img width="690" alt="Capture d’écran 2021-08-30 à 09 20 30" src="https://user-images.githubusercontent.com/119625/131301241-5c15ffec-afbc-4919-8ad1-838fa1c81bfe.png">

Reply:
Ok then. 
I'll be more productive in terms of features.
We lost the button on the new version tho I've open an issue : https://github.com/etalab/data.gouv.fr/issues/476
This functionnality is also interesting to gather "good" reuses. Is there already an easy way to get all featured reuses (other than exporting the catalog ?)

Reply:
For context @maudetes 
![image](https://user-images.githubusercontent.com/60264344/131312322-e54f4f7e-e139-4477-8aea-ed9c7b87b42c.png)
![image](https://user-images.githubusercontent.com/60264344/131312389-9b6f3177-aeab-47a1-ad57-b1fd0908bafb.png)

Reply:
I think https://www.data.gouv.fr/fr/reuses/?featured=true will do what you ask. But I find it weird that the tab doesn't show the latest reuses if called `latest reuses` though.
(When I look at https://www.data.gouv.fr/fr/reuses/?sort=-created, the results look very clean)

Reply:
You need to go there _before_ I clean up :-) I agree there's a discrepancy between "latest" and the feature, but let's call it moderation _a posteriori_.

Reply:
I kept the `featured` filter and we can prioritize https://github.com/etalab/data.gouv.fr/issues/476 for next sprints (could work for *refonte* or *decouvrability* topics).

Reply:
Perfect thanks !